GrandCentralBoard/Widgets/Harvest/HarvestWidgetView.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  HarvestWidgetView.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import UIKit
+
+
+class HarvestWidgetView: UIView {
```

Comment:
I think this name change should be done within the UI ticket instead. As far as I remember from the designs, the view shows more than one chart, so `SimpleBarChart` may not be appropriate.